rotkehlchen/tests/api/test_database.py

```diff
@@ -118,7 +118,7 @@ def test_delete_download_backup_errors(rotkehlchen_api_server, data_dir, usernam
         api_url_for(
             rotkehlchen_api_server,
             'databasebackupsresource'),
-        json={'file': str(Path(user_data_dir, 'idontexist'))},
+        json={'files': [str(Path(user_data_dir, 'idontexist'))]},
```

Comment:
You need to add a test for this. You need a test where:

1. Multiple backups are deleted
2. One backup can be deleted, other backup can't and then the whole call fails WITHOUT deleting any of them